tests/psmdbcluster/psmdb_cluster_test.go

```diff
@@ -85,6 +88,36 @@ func TestPSMDBClusterAPI(t *testing.T) {
 	}
 	assert.True(t, clusterFound)
 
+	t.Log("Wating for cluster to be ready")
+	err = waitForPSMDBClusterState(tests.Context, kubeconfig, name, controllerv1beta1.PSMDBClusterState_PSMDB_CLUSTER_STATE_READY)
+	require.NoError(t, err)
+
+	updateReq := &controllerv1beta1.UpdatePSMDBClusterRequest{
+		KubeAuth: &controllerv1beta1.KubeAuth{
+			Kubeconfig: kubeconfig,
+		},
+		Name: name,
+		Params: &controllerv1beta1.PSMDBClusterParams{
+			ClusterSize: 6,
+			Replicaset: &controllerv1beta1.PSMDBClusterParams_ReplicaSet{
+				ComputeResources: &controllerv1beta1.ComputeResources{},
```

Comment:
let's set some values here to be sure that we are updating compute resources

Reply:
same comment as the previous. there is no much I can do before running out of resources.

Reply:
how much of cpu do you have? we can set cpu for 500m and try.